edge-rtac/src/main/resources/domain/edge-rtac/features/rtac.feature

```diff
@@ -0,0 +1,148 @@
+    * def createFirstItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem') { itemId: 69640328-788e-43fc-9c3c-af39e243f3b7 }
+    * def itemStatusName = 'Checked out'
+    * def createSecondItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem') { itemId: 79640328-788e-43fc-9c3c-af39e243f3b7 }
+
+    Given path 'rtac/' + instanceId
+    When method GET
+    Then status 200
+    And match response.holdings..id contains [69640328-788e-43fc-9c3c-af39e243f3b7, 79640328-788e-43fc-9c3c-af39e243f3b7]
+    And match response.holdings..status contains ['Available' , 'Checked out']
+
+  Scenario: For a non-periodical/non-serial, return holdings and item information including availability for each instance UUID included in request
+#   1st instance
+    * def instanceId = call random_uuid
+    * def servicePointId = call random_uuid
+    * def locationId = call random_uuid
+    * def holdingId = call random_uuid
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def firstInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def createFirstItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem') { itemId: 68640328-799e-43fc-9c3c-af39e243f3b7 }
+
+#   2nd instance
+    * def instanceId = call random_uuid
+    * def servicePointId = call random_uuid
+    * def locationId = call random_uuid
+    * def holdingId = call random_uuid
+    * def createInstanceResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostInstance')
+    * def secondInstanceId = createInstanceResponse.instanceEntityRequest.id
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostLocation')
+    * call read('classpath:domain/edge-rtac/features/util/initData.feature@PostHoldings')
+    * def itemStatusName = 'Checked out'
+    * def createSecondItemResponse = call read('classpath:domain/edge-rtac/features/util/initData.feature@PostItem') { itemId: 68640328-788e-53fc-9c3c-af39e243f3b7 }
+    * def periodicalStatus = true
+    * def rtacBatchrequest = read('samples/rtac-batch-request-entity.json')
+
+    Given path 'rtac-batch'
+    And request rtacBatchrequest
+    When method POST
+    Then status 200
+    And match response.holdings..id contains [ 68640328-799e-43fc-9c3c-af39e243f3b7 , 68640328-788e-53fc-9c3c-af39e243f3b7 ]
+    And match response.holdings..status contains [ Available , Checked out ]
```

Comment:
Does that match applying ordering?

If it doesn't, is it important that the status is for the right holdings?

Reply:
api response is get shuffled for each test run. 
And i think test is just for checking weather its returning status & detail or not.

Reply:
> api response is get shuffled for each test run.

What do you mean by shuffled?

> And i think test is just for checking weather its returning status & detail or not.

I think the test should be specific that each holding has the expected status